packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,280 @@
+                }))
+            })
+    }
+
+    render() {
+        const { model, onInputChange, onCategoryChange } = this.props;
+        const { availableCategories } = this.state;
+
+        return (
+            <>
+                <SectionHeading title="Basic Properties" />
+
+                <DatasetSettingsInput
+                    name="name"
+                    label="Name"
+                    helpTip={this.getHelpTipElement("name")}
+                    value={model.name}
+                    placeholder="Enter a name for this dataset"
+                    disabled={false}
+                    onValueChange={onInputChange}
+                    showInAdvancedSettings={false}
+                    required={true}
+                />
+
+                <DescriptionInput
+                    model={model}
+                    onInputChange={onInputChange}
+                />
+
+                <Row className={'margin-top'}>
+
+                    <Col xs={4}>
+                        <DomainFieldLabel
+                            label={"Category"}
+                            required={false}
+                            helpTipBody={() => this.getHelpTipElement("category")}
+                        />
+                    </Col>
+
+                    <Col xs={7}>
+                        {/*TODO: could be replaced by QuerySelect along with server side story*/}
+                        <Creatable
+                            name="categoryId"
+                            placeholder="Select dataset category"
+                            onChange={onCategoryChange}
+                            value={model.categoryId}
+                            options={availableCategories}
+                        />
+                    </Col>
+
+                    <Col xs={1}/>
+
+                </Row>
+
+                <DatasetSettingsInput
+                    name="label"
+                    label="Label"
+                    helpTip={this.getHelpTipElement("label")}
+                    value={model.label}
+                    disabled={false}
+                    onValueChange={onInputChange}
+                    showInAdvancedSettings={false}
+                    required={false}
+                />
+
+            </>
+        );
+    }
+}
+
+interface DataRowUniquenessElementsProps {
+    onRadioChange: (evt: any) => any;
+    dataRowSetting: number;
+}
+
+class DataRowUniquenessElements extends React.PureComponent<DataRowUniquenessElementsProps> {
+    render() {
+        const { onRadioChange, dataRowSetting } = this.props;
+        const radioName = "dataRowSetting";
+
+        return (
+            <div className='dataset_data_row_uniqueness_container'>
+                <Radio
+                    name={radioName}
+                    value={0}
+                    checked={dataRowSetting == 0}
+                    onChange={onRadioChange}
+                >
+                    Participant ID only (demographic data)
```

Comment:
We don't have to do this now, but we will want to use the given containers study module context subject information for this text (i.e. instead of hardcoding in "Participant" here we'd use LABKEY.moduleContext.study.subject)